lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/MessageUtils.java

```diff
@@ -99,9 +99,19 @@ public static void copyDBOToDTO(DBOMessageToUser info, MessageToUser bundle) {
 		bundle.setWithUnsubscribeLink(info.getWithUnsubscribeLink());
 		bundle.setWithProfileSettingLink(info.getWithProfileSettingLink());
 		bundle.setIsNotificationMessage(info.getIsNotificationMessage());
-		bundle.setTo(info.getTo());
-		bundle.setCc(info.getCc());
-		bundle.setBcc(info.getBcc());
+		try {
+			if (info.getTo() != null) {
```

Comment:
what if info.getTo()==null?  Should this code then call bundle.setTo(null)?

Reply:
I made the assumption that the bundle passed in had all null fields, but you're right, it's better to explicitly set the field to null.